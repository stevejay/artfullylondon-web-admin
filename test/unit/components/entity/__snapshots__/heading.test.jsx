import React from 'react'
import { shallow } from 'enzyme'

import EntityHeading from '_src/components/entity/heading'

it('should render correctly', () => {
  const wrapper = shallow(
    <EntityHeading>
      <div id='child' />
    </EntityHeading>
  )

  expect(wrapper).toMatchSnapshot()
})
