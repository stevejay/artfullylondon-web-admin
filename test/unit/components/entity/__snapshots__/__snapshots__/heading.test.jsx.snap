// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<h1
  className="components-entity-Heading-m__heading"
>
  <div
    id="child"
  />
</h1>
`;
